cabal.project

```diff
@@ -202,8 +202,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 631cb6cf1fa01ab346233b610a38b3b4cba6e6ab
-  --sha256: 0944wg2nqazmhlmsynwgdwxxj6ay0hb9qig9l128isb2cjia0hlp
+  tag: 0f3a867493059e650cda69e20a5cbf1ace289a57
```

Comment:
Looks good: 0f3a867493059e650cda69e20a5cbf1ace289a57 is one commit ahead of 631cb6cf1fa01ab346233b610a38b3b4cba6e6ab